buildenv/jenkins/variables/defaults.yml

```diff
@@ -300,7 +298,6 @@ x86-64_linux:
       # use default gcc for 8, 11 (v7.5)
       8: ''
       11: ''
-      17: 'source /home/jenkins/set_gcc_10.3.0_env'
```

Comment:
I think `x86-64_linux_criu` should inherit all of `build_env` from here so it will use the came compiler versions (i.e. delete lines 518-523).

Reply:
Updated.